### ISSUE TITLE
Download vs Install need documentation

### ISSUE THREAD

**Issue by mcp292** [syl20bnr/spacemacs#15240]
On the main site you are given the option to download or install, but these options are not documented. I checked the FAQ and found nothing and the quick start and documentation pages don't mention it.

By chance I've discovered that "Download" gets you the `master` version and "Install" gets you the `develop` version. The names "download" and "install" are not self-documenting. Perhaps "stable" and "develop" would be better options.

**Comment by mcp292** [comment]
I've found the [installation guide](https://github.com/syl20bnr/spacemacs#install) on the GitHub page, but it is not linked to from the website, spacemacs.org, which was my starting point and introduction to this program.

In that guide it mentions that the master method is deprecated, so shouldn't new users be encouraged to go the "Install" route? 

Further, if rolling release is the new standard, shouldn't the "Update Spacemacs" button on the splash screen do `git pull --rebase` for you, instead of mentioning in the error that you are on `develop`, which insinuates you've diverged from the recommended route?

**Comment by lebensterben** [comment]
The inconsistency of `download` vs `install` is confirmed.

---

> Further, if rolling release is the new standard,

We have not reached such standard yet.

---

> shouldn't the "Update Spacemacs" button on the splash screen do git pull --rebase for you, instead of mentioning in the error that you are on develop, which implies you've diverged from the recommended route

This is a design choice.
One should be well-aware of the potential instability on running on a `develop` branch.
The warning message helps to refresh such awareness.

**Comment by mcp292** [comment]
> One should be well-aware of the potential instability on running on a `develop` branch. The warning message helps to refresh such awareness.

Okay that brings us full circle then. Can you expand on

>The inconsistency of `download` vs `install` is confirmed.

**Comment by lebensterben** [comment]
What do you expect me to expand to?

**Comment by mcp292** [comment]
> is confirmed

By you, or is documented in an issue elsewhere... Please expand.

**Comment by lebensterben** [comment]
The inconsistency is confirmed and I already said that.

There's nothing to expand.

**Comment by mcp292** [comment]
By you, or is documented in an issue elsewhere...

**Comment by lebensterben** [comment]
What do you mean by "by you"?

I've repeatedly recognized the issue.

**Comment by mcp292** [comment]
Let me rephrase: When you say "the inconsistency of download vs install is confirmed" are you saying that you are confirming the inconsistency as of that statement, or that (since you use past tense "is confirmed") it has been confirmed or acknowledged already elsewhere in the project.

**Comment by lebensterben** [comment]
I don't want to teach you English.

"is confirmed" is not past tense.
It simple present tense + passive voice.

**Comment by mcp292** [comment]
Great, no answer. Why are you participating in an open source project? It obviously doesn't suit your attitude.

**Comment by lebensterben** [comment]
Confirming a it's bug is just the first step in solving the issue.

It's meant to be a marker, as long as the tags I added, for other contributors so they don't need to repeat this step.

Is it clear to you now?

**Comment by mcp292** [comment]
I'm not sure how you propose to teach English when you couldn't comprehend and answer [a simple, clearly stated question](https://github.com/syl20bnr/spacemacs/issues/15240#issuecomment-1007562733).

The real issue though, is you've chosen to put your energy towards contending me from [the outset of my participation](https://github.com/syl20bnr/spacemacs/issues/15232), instead of putting that energy towards solving the issues or at the very least defining a clear pathway to do so.

I can't help but state the observation that you are in the wrong hobby/profession. You are hurting the project with your lack of productivity and through your active deterrence of contribution, both to me directly, but also by sending a strong message (of your own weakness) to any onlookers.

**Comment by lebensterben** [comment]
See https://github.com/syl20bnr/spacemacs.org/issues/37 for inconsistency of download and install button.
